pusher-api/overview.md

```diff
@@ -6,10 +6,11 @@ With subscriptions, the Pusher API will immediately update you of those events t
 * [Plan subscriptions](#plan-subscriptions)
 * [Manage subscriptions](#manage-subscriptions)
 
+
 ## Understand how subscriptions work
 The Pusher API provides events for you to listen to certain activities of the Zettle Go app at a working HTTPS endpoint on your server.
 
-After you subscribe to events, when an event is triggered, the Pusher API sends a `POST` request that contains a `payload` field. The field contains event information to the HTTPS endpoint in real time.
+After you subscribe to events, when an event is triggered, the Pusher API sends the event in a `POST` request to the HTTPS endpoint. The request contains a `payload` field with event information in real time.
```

Comment:
Long sentence, suspect Hemingway reader doesn't like it :-), but don't have much of a suggestion. How about:
"When subscribing to events and an event is triggered, the Pusher API sends a `POST` request to the HTTPS endpoint. The request contains a `payload` field with event information in real time."

Reply:
Thanks! I ran it and an updated text based on your suggestion in Hemingway.Still too long, but it reads better.